CONTRIBUTING.md

```diff
@@ -1,19 +1,17 @@
-## Welcome!
+# Welcome!
 
-We're so glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
-
-We want to ensure a welcoming environment for all of our projects. Our staff follow the [18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md) and all contributors should do the same.
+We're so glad you're thinking about contributing to a [Federal Government open source project](https://code.gov/)! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
```

Comment:
I don't see a serious issue with generalizing the verbiage here, but `Federal Government` is a bit too formal. I would prefer to see `federal open source project` or `open source project of the U.S. government`. I do have some reservations about appearing to represent the entire government, and would maybe prefer to keep it as 18F or TTS, but don't feel strongly.